ui/js/models/job_log_url.js

```diff
@@ -30,8 +30,10 @@ treeherder.factory('ThJobLogUrlModel', [
                 .then(function(response) {
                     var item_list = [];
                     angular.forEach(response.data, function(elem){
-                        var buildUrl = elem.url.slice(0, elem.url.lastIndexOf("/")) + "/";
-                        elem.buildUrl = buildUrl;
+                        if(elem.name === "buildbot_text" || elem.name === "errorsummary_json") {
```

Comment:
Drive by :) I guess there could be a space between the `if` and the open paren here.